test/unit/builtins/math.spec.ts

```diff
@@ -24,3 +24,17 @@ test.each(["E", "LN10", "LN2", "LOG10E", "LOG2E", "SQRT1_2", "SQRT2"])("Math.%s"
         expect(builder.getLuaExecutionResult()).toBeCloseTo(builder.getJsExecutionResult());
     });
 });
+
+const expectMathAtan2: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toContain("math.atan2(");
+const expectMathAtan: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toContain("math.atan(");
+
+util.testEachVersion("Math.atan2", () => util.testExpression`Math.atan2(4, 5)`, {
+    [tstl.LuaTarget.LuaJIT]: builder => builder.tap(expectMathAtan2),
+    [tstl.LuaTarget.Lua51]: builder => builder.tap(expectMathAtan2),
+    [tstl.LuaTarget.Lua52]: builder => builder.tap(expectMathAtan2),
+    [tstl.LuaTarget.Lua53]: builder => builder.tap(expectMathAtan),
+});
+
+test("Lua 5.3 Math.atan2 comparison", () => {
+    util.testExpression`Math.atan2(4, 5)`.setOptions({ luaTarget: tstl.LuaTarget.Lua53 }).expectToMatchJsResult();
```

Comment:
5.3 is a default target

```suggestion
    util.testExpression`Math.atan2(4, 5)`.expectToMatchJsResult();
```